Mathlib/Condensed/TopComparison.lean

```diff
@@ -30,8 +30,8 @@ universe w w' v u
 
 open CategoryTheory Opposite Limits regularCoverage ContinuousMap
 
-variable {C : Type u} [Category.{v} C] (G : C ⥤ TopCat.{v})
```

Comment:
I don't think this is necessarily more general; omitting `.{}` means that a universe metavariable is used instead, which could potentially be populated with something *less* general. Could you keep the explicit universe names here (perhaps using more of them) to ensure the generality is what you intend?

Reply:
In this case it is completely general. I've added explicit names for the universes.